covid_spike_classification/__main__.py

```diff
@@ -47,7 +49,9 @@ def main():
         if config.zip_results:
             shutil.make_archive(config.outdir, "zip", root_dir=config.outdir)
             shutil.rmtree(config.outdir, ignore_errors=True)
-
+        if config.stdout:
+            outfile = open(os.path.join(config.outdir, "results.csv"), "r")
```

Comment:
When running with `--zip-results`, this will fail, as line 51 removes the `config.outdir`. Maybe just move this `if` statement above the other one.

Reply:
Good point - will update!